docs/data/joy/components/skeleton/BasicSkeleton.tsx

```diff
@@ -0,0 +1,26 @@
+import * as React from 'react';
+import AspectRatio from '@mui/joy/AspectRatio';
+import Card from '@mui/joy/Card';
+import Skeleton from '@mui/joy/Skeleton';
+import Typography from '@mui/joy/Typography';
+
+export default function BasicSkeleton() {
+  return (
+    <Card variant="outlined" sx={{ width: 343, display: 'flex', gap: 2 }}>
+      <AspectRatio ratio="21/9">
+        <Skeleton variant="overlay">
+          <img
+            alt=""
+            src="https://images.unsplash.com/photo-1686548812883-9d3777f4c137"
```

Comment:
This image is huge: 4 MB, fixed in 08adce666bf7058e0883803b601d619906a4fd4a

Reply:
But I don't understand this demo. The img doesn't make sense to me, it's loading, by definition we don't have the image yet. Why is it not?

```suggestion
            src=""
```

Reply:
Works for me.